src/components/transactions/delegateStatistics.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mountWithContext } from '../../../test/utils/mountHelpers';
+import DelegateStatistics from './delegateStatistics';
+
+describe('DelegateStatistics', () => {
+  const storeState = {
+    account: {
+      votes: new Array(40).fill({
+        username: null,
+        address: '3484156157234038617L',
+        publicKey: 'bd56ce59f413370cf45dbc4be094acbd4de9c6894443476e5406dfc458337889',
+        balance: '0',
+      }),
+    },
+  };
+
+  it('should render DelegateStatistics', () => {
+    const wrapper = mountWithContext(<DelegateStatistics />, {});
+    expect(wrapper.find('DelegateStatistics').length).to.have.equal(1);
+  });
+
+  it('should not display showAll button', () => {
```

Comment:
can you please be a bit more specific about what this is testing? it is confusing to read  that it should not display the button, when initially we expect it to be shown (which you are testing). e.g. 
`should not show the "show more" button, if all votes are already shown`